### DISCUSSION TITLE
Audio files preview feature requests

### DISCUSSION THREAD

**Discussion by rrrromanov** [doublecmd/doublecmd#1337] [Suggestions]
MacOS, MacPreview
Would be great to have:
- autoplay feature. I need to check 10 files, I have to start them all manually upon opening in view
- remaining time feature. In quick view it is unclear, how long the audio is
- misc info: bitrate, codec, dicretization etc.

**Comment by Skif-off** [comment]
Win: [Windows Media Player](https://doublecmd.github.io/doc/en/plugins.html), [Mmedia](https://totalcmd.net/plugring/mmedia.html), maybe [Explorer Preview](https://doublecmd.github.io/doc/en/plugins.html). (Also TotalAmp and vLister, but these plugins are a little hard to find.)
macOS: [MacPreview](https://doublecmd.github.io/doc/en/plugins.html).
Linux: [gstplayer (GTK2), wlxmpv (Qt5/Qt6)](https://github.com/doublecmd/doublecmd/wiki/Plugins#user-content-gstplayer), [gstplayer-mod (GTK2), mpv (GTK2)](https://github.com/j2969719/doublecmd-plugins/blob/master/plugins.md#wlxgtk), [qmediaplayer_qt (Qt5/Qt6)](https://github.com/j2969719/doublecmd-plugins/blob/master/plugins.md#wlxqt5), [WlxMplayer GTK2, Qt5, Qt6](https://doublecmd.github.io/doc/en/plugins.html).

**Reply by rrrromanov** [reply]
Was not clear, added details. My suggestion was on MacPreview.

**Reply by rich2014** [reply]
`MacPreview` uses the system's native function `QuickLook`, which does not provide such fine control, so it cannot be implemented in MacPreview.

**Comment by rich2014** [comment]
@rrrromanov 

done.

download from https://doublecmd.sourceforge.io/site/eng/snapshots.php and test it.
